test/PoolFactory/deployPool.spec.ts

```diff
@@ -265,4 +270,127 @@ describe("PoolFactory.deployPool", () => {
+            const oldPoolAddress = await factory.pools(0)
+            const oldPool = (await ethers.getContractAt(
+                "LeveragedPool",
+                oldPoolAddress
+            )) as LeveragedPool
+            const newPoolAddress = await factory.pools(poolNumber)
+            const newPool = (await ethers.getContractAt(
+                "LeveragedPool",
+                newPoolAddress
+            )) as LeveragedPool
+
+            const oldFeeSplit = (
+                await oldPool.secondaryFeeSplitPercent()
+            ).toNumber()
+            const newFeeSplit = (
+                await newPool.secondaryFeeSplitPercent()
+            ).toNumber()
+            expect(oldFeeSplit).to.eq(10)
+            expect(newFeeSplit).to.eq(20)
+        })
+
+        it("pool fee transfers new fee split percentage", async () => {
+            const deploymentData = {
+                poolName: POOL_CODE_2,
+                frontRunningInterval: 3,
+                updateInterval: 5,
+                leverageAmount: 2,
+                quoteToken: token.address,
+                oracleWrapper: oracleWrapper.address,
+                settlementEthOracle: settlementEthOracle.address,
+            }
+
+            const poolNumber = (await factory.numPools()).toNumber()
+            await factory.deployPool(deploymentData)
+            const newPoolAddress = await factory.pools(poolNumber)
+            const newPool = (await ethers.getContractAt(
+                "LeveragedPool",
+                newPoolAddress
+            )) as LeveragedPool
+
+            let commiter = await newPool.poolCommitter()
+            const poolCommitter = (await ethers.getContractAt(
+                "PoolCommitter",
+                commiter
+            )) as PoolCommitter
+
+            // Make commits
+            await token.approve(
+                newPool.address,
+                ethers.utils.parseEther("10000")
+            )
+            await createCommit(
+                poolCommitter,
+                LONG_MINT,
+                ethers.utils.parseEther("2000")
+            )
+            await createCommit(
+                poolCommitter,
+                SHORT_MINT,
+                ethers.utils.parseEther("2000")
+            )
+            const signers = await ethers.getSigners()
+            await newPool.setKeeper(signers[0].address)
+            newPool.updateSecondaryFeeAddress(secondFeeAddress)
+
+            // Execute commits no price change
+            await timeout(updateInterval * 1000)
+            const lastPrice = ethers.utils.parseEther(
+                getRandomInt(99999999, 1).toString()
+            )
+            await newPool.poolUpkeep(lastPrice, lastPrice)
+
+            // Upkeep pool with price change
+            await timeout(updateInterval * 1000)
+            await newPool.poolUpkeep(
+                lastPrice,
+                BigNumber.from("2").mul(lastPrice)
+            )
```

Comment:
I think this should be in a hook of some description.

Reply:
you mean in a before or beforeAll? I didn't do that because it isn't needed by any of the other test cases so its kinda a one of.